apache/zeppelin
# [Zeppelin 914]- Apply new mechanism to IgniteInterpreter

What is this PR for?

This handles replacing the registration of interpreter with static block by the interpreter-setting.json file

What type of PR is it?

[| Improvement |

Todos

Sub-Task

What is the Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-914

How should this be tested?

There shouldn't be any warning like below on starting the server
INFO 2016-09-29 00:25:46,247 - Bootstrapping Ignite Interpreter WARN 2016-09-29 00:25:46,250 - Static initialization is deprecated for interpreter Ignite, You should change it to use interpreter-setting.json in your jar or interpreter/{interpreter}/interpreter-setting.json INFO 2016-09-29 00:25:46,250 - Inclass=org.apache.zeppelin.ignite

And ensure that the Ignite related paragraphs run without any error

Screenshots (if appropriate)

Questions:

Does the licenses files need update? No
Is there breaking changes for older versions? No
Does this needs documentation? No
